test/commands/apps/info.js

```diff
@@ -241,12 +241,17 @@ stack=cedar-14
       .get('/apps/myapp/addons').reply(200, addons)
       .get('/apps/myapp/collaborators').reply(200, collaborators)
       .get('/apps/myapp/dynos').reply(200, [{type: 'web', size: 'Standard-1X', quantity: 2}])
+      .get('/apps/myapp/pipeline-couplings').reply(200, { app: { id: appAcm.id }, pipeline: { name: 'my-pipeline' } })
     return cmd.run({args: {app: 'myapp'}, flags: {json: true}})
       .then(() => expect(cli.stderr).to.equal(''))
       .then(() => {
         let json = JSON.parse(cli.stdout)
         expect(json.appExtended).to.equal(undefined)
         expect(json.app.extended).to.equal(undefined)
+        expect(json.addons.length).to.equal(addons.length)
+        expect(json.collaborators.length).to.equal(collaborators.length)
+        expect(json.dynos[0].type).to.equal('web')
```

Comment:
These 3 extra lines is to check the existence of these other keys, which we weren't testing for before.